files/demo.hledger

```diff
@@ -0,0 +1,178 @@
+﻿# Welcome to hledger, This file is where your financial history will be recorded.
+# It is a plain text file, so it is easy to edit and manipulate, and you can use
+# search and replace functionality in your text editor
+
+# Everything after '#', ';' or '*' are comments, so they are not read by hledger,
+# except in some cases to be discussed later
```

Comment:
FWIW `;` works in more places, eg after other content. `#` and `*` only work at the start of the line.

You jump back and forth between all three below, maybe it's a little bit distracting ? I would use # (or ; ) for file comments, and avoid using * unless you are showing how it's useful for org/orgstruct mode in emacs.

Reply:
I didn't know about that. I prefer to just use ';' so I can avoid explaining this difference, ok?